content/en/monitors/monitor_types/anomaly.md

```diff
@@ -26,7 +26,7 @@ For example, anomaly detection can help you discover when your web traffic is un
 
 To create an [anomaly monitor][1] in Datadog, use the main navigation: *Monitors --> New Monitor --> Anomaly*.
 
-The anomaly detection monitor, use the `anomalies` function from the Datadog query language. When you apply this function to a series, it returns the usual results along with an expected "normal" range.
+The anomaly detection monitor uses the `anomalies` function from the Datadog query language. When you apply this function to a series, it returns the usual results along with an expected "normal" range.
```

Comment:
Is this line useful? I wouldn't mind seeing this part dropped here and in the forecast monitor doc.

Reply:
I can remove these.